mirage/views/external-account.ts

```diff
@@ -0,0 +1,30 @@
+import { HandlerContext, ModelInstance, Request, Schema } from 'ember-cli-mirage';
+
+import ExternalAccountsModel from 'ember-osf-web/models/external-accounts';
+import { process } from './utils';
+
+export function externalAccountDetail(this: HandlerContext, schema: Schema, request: Request) {
+    const { accountID } = request.params;
+    const externalAccount = schema.externalAccounts.find(accountID);
+
+    const serializedExternalAccount = this.serialize(externalAccount);
+    const { data } = process(schema, request, this, [serializedExternalAccount.data]);
+
+    return {
+        data: data[0],
+        meta: serializedExternalAccount.meta,
+    };
+}
+
+export function externalAccountList(this: HandlerContext, schema: Schema, request: Request) {
+    const userAddons = schema.users
+        .find(request.params.userID)['userAddons']
+        .models;
+    const externalAccounts = userAddons.length ? userAddons[0].externalAccounts.models : null;
+    if (externalAccounts) {
+        externalAccounts.map((model: ModelInstance<ExternalAccountsModel>) => this.serialize(model).data);
```

Comment:
I think this needs an additional filter on it to filter by provider. I was so happy I got it working that I forgot that step.